packages/test-studio/src/sidecar/index.js

```diff
@@ -0,0 +1,29 @@
+// Add this to sanity.json if you want confetti in the sidecar
+// {
+//   "implements": "part:@sanity/default-layout/sidecar",
+//   "path": "src/sidecar/index.js"
+// },
+
+/* eslint-disable react/no-multi-comp */
+import React from 'react'
+import ToggleSidecarButton from 'react-icons/lib/ti/weather-snow'
+import {toggleSidecarOpenState} from 'part:@sanity/default-layout/sidecar-datastore'
+import Confetti from 'react-confetti'
+
+export function isSidecarEnabled() {
```

Comment:
What's the purpose of this and could it be a const instead of a function?

Reply:
Got rid of the whole folder, actually :)